modules/theme-tailwind-shopware/cypress/support/pageObjects/category/visitCategory.js

```diff
@@ -0,0 +1,3 @@
+export default function visitCategory(category) {
```

Comment:
How is this different from hypothetical visitProduct? Why not general `visit` helper function?